[2022-10-01 19:37:43,283] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: main_dag.store_fct_table manual__2022-10-01T19:36:24.220735+00:00 [queued]>
[2022-10-01 19:37:43,308] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: main_dag.store_fct_table manual__2022-10-01T19:36:24.220735+00:00 [queued]>
[2022-10-01 19:37:43,314] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-01 19:37:43,320] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-01 19:37:43,321] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-01 19:37:43,387] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): store_fct_table> on 2022-10-01 19:36:24.220735+00:00
[2022-10-01 19:37:43,401] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'store_fct_table', 'manual__2022-10-01T19:36:24.220735+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpbhi4som5', '--error-file', '/tmp/tmpuxkxvxjf']
[2022-10-01 19:37:43,404] {standard_task_runner.py:80} INFO - Job 49: Subtask store_fct_table
[2022-10-01 19:37:43,397] {standard_task_runner.py:52} INFO - Started process 130 to run task
[2022-10-01 19:37:43,564] {task_command.py:369} INFO - Running <TaskInstance: main_dag.store_fct_table manual__2022-10-01T19:36:24.220735+00:00 [running]> on host cb37a969dddb
[2022-10-01 19:37:43,680] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=main_dag
AIRFLOW_CTX_TASK_ID=store_fct_table
AIRFLOW_CTX_EXECUTION_DATE=2022-10-01T19:36:24.220735+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-01T19:36:24.220735+00:00
[2022-10-01 19:37:43,682] {postgres.py:127} INFO - Running copy expert: COPY fct_table FROM stdin WITH DELIMITER as ',', filename: df_fct_table.csv
[2022-10-01 19:37:43,694] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-10-01 19:37:43,708] {python.py:173} INFO - Done. Returned value was: None
[2022-10-01 19:37:43,732] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=store_fct_table, execution_date=20221001T193624, start_date=20221001T193743, end_date=20221001T193743
[2022-10-01 19:37:43,790] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-01 19:37:43,945] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
